core/meterer/metering_store.go

```diff
@@ -27,8 +27,9 @@ type MeteringStore interface {
 	// RollbackOnDemandPayment rolls back a payment to the previous value
 	RollbackOnDemandPayment(ctx context.Context, accountID gethcommon.Address, newPayment, oldPayment *big.Int) error
 
-	// GetPeriodRecords fetches period records for the given account ID and reservation period
-	GetPeriodRecords(ctx context.Context, accountID gethcommon.Address, reservationPeriod uint64) ([MinNumBins]*pb.PeriodRecord, error)
+	// GetPeriodRecords fetches period records for the given account ID and reservation period across multiple quorums
+	// Consider alternative: singular quorumId and reservationPeriod as an input, require multiple calls
```

Comment:
nit: Is the comment here about the alternative needed?

Reply:
this is like an alternative replacement. Since this API might be used by DataAPI, I was curious if they would like an alternative type

Reply:
this comment is just considering if other clients, specificially DataAPI, want a different interface; or if other engineers would think the alternative is more performant/optimized in some situations. the one defined here is suited for how disperser would use it

Reply:
Ah I see, but I guess the comment feels more like a todo to validate if that call pattern is sufficient for DataAPI rather than documentation about how GetPeriodRecords works?

Reply:
hmm yeah it is a todo. maybe @pschork can answer this? I can update within this PR to accommodate the usage

Reply:
I think dataAPI can work with current multi quorum results.